### PR TITLE
Add fee collectors management how-to

### PR DESCRIPTION
**Pull request by TucksonDev**
This PR adds a page that explains how to manage the different fee collector addresses on an Orbit chain.

To do:

- [x] Add "how to use the distribution contracts"

[Preview](https://arbitrum-docs-git-feat-add-fee-collectors-b43155-offchain-labs.vercel.app/launch-orbit-chain/how-tos/manage-fee-collectors)

### CODE REVIEW

**Review by symbolpunk — APPROVED** [review]
No blocking feedback from my perspective. Approving but will hold off on merging in case your intent is to merge after SME approval.

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:16]:
```suggestion
This guide describes the different fees that are collected, and explains how to specify the fee collector address on your chain for each fee type.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:14]:
```suggestion
As part of the activity of an Orbit chain, different fees are collected with every transaction. These fees are collected as a single amount (the transaction fees) but are internally split into different components depending on their purpose. Each component can also be transferred to a different fee collector address that can be configured on your chain.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:20]:
```suggestion
There are four fee types that are collected on every transaction of an Orbit chain:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:22]:
```suggestion
- **Orbit base fee**: fees paid for executing transactions on the chain based on the minimum base price configured.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:24]:
```suggestion
- **Orbit surplus fee**: if the chain is congested (i.e., the base price paid for the transaction is higher than the minimum base price), these fees account for executing transactions on the chain based on any gas price paid above the minimum base price configured.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:26]:
```suggestion
- **Parent chain base fee**: relative fees paid for posting the transaction on the parent chain. The amount is calculated based on the transaction's estimated size and the current view of the parent chains' base fee.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:37]:
```suggestion
Let's now look at how to configure the collector addresses for each fee type.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:41]:
```suggestion
Orbit base fees are paid to the infraFeeAccount configured in your chain. You can retrieve the current address by calling the getInfraFeeAccount() method of the [ArbOwnerPublic](https://arbitrum-docs-igdj6umeq-offchain-labs.vercel.app/build-decentralized-apps/precompiles/reference#arbownerpublic) precompile. For example:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:91]:
```suggestion
Orbit surplus fees are paid to the `networkFeeAccount` configured in your chain. You can retrieve the current address by calling the method `getNetworkFeeAccount()` of the [ArbOwnerPublic](/build-decentralized-apps/precompiles/02-reference.md#arbownerpublic) precompile. For example:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:99]:
```suggestion
Alternatively, you can use the Orbit SDK to retrieve the current address as `networkFeeAccount`, by calling the [ArbOwner](/build-decentralized-apps/precompiles/02-reference.md#arbowner) precompile:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:143]:
```suggestion
Reading information from the `ArgAggregator` precompile or using it to set new information is currently not supported in the Orbit SDK but will be added soon. So, for now, this subsection will only show examples using `cast call` and `cast send`.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:159]:
```suggestion
Once you have the current batch poster, you can obtain the fee collector address configured for that batch poster by calling the method `getFeeCollector(address)` of the [ArbAggregator](/build-decentralized-apps/precompiles/02-reference.md#arbaggregator) precompile and passing the address of the batch poster.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:270]:
```suggestion
For now, we've described how to set the individual collector addresses for each fee type. Some chains may require multiple addresses to receive the collected fees of any of the available types. In those cases, there's the possibility of using a distributor contract that can gather all fees of a specific type and distribute those among multiple addresses.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:276]:
```suggestion
Currently, the Orbit SDK doesn't support deploying and configuring distribution contracts, but it will soon be added. So, for now, this section will only show examples using `cast send`.
```

File: arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md
Comment (anegg0) [inline arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md:292]:
```suggestion
For example, if you want to set two addresses as receivers, with the first one receiving 80% of the fees and the second one receiving 20% of the fees, you'll use the following parameters:
```

**Review by anegg0 — COMMENTED** [review]
It's all nice and clear!
I just added nit-picks, which are optional.

**Comment by TucksonDev** [comment]
Thanks @anegg0 !
I've added the suggested changes 🙏

**Pull request merged**
